src/helpers/GetCorrectUnits.ts

```diff
@@ -0,0 +1,68 @@
+import { DecimalBigNumber } from "./DecimalBigNumber/DecimalBigNumber";
```

Comment:
Consider shifting this into a helpers directory underneath Give, or add give to the filename.

Reply:
Going to move this one into a helpers folder and the helpers that just pull data like GetDonationDate, I'm going to rename with Give included (until they all are moved to a hook for react query)